libs/architectures/mlpe/architectures/embeddings/convolution.py

```diff
@@ -102,3 +102,39 @@ def get_flattened_size(self, size: int):
         out = self.downsampler(x)
         out = torch.flatten(out)
         return len(out)
+
+
+class NaiveEmbedding(torch.nn.Module):
```

Comment:
This is great. Can we put this in its own file and maybe call the class something like `FullyConnectedEmbedding` or `DenseEmbedding`?

Reply:
Done